pkg/course/course.go

```diff
@@ -258,10 +271,10 @@ func OpenCourseV2(fileName string, schema []byte) (*FileV2, error) {
 	if err != nil {
 		return nil, err
 	}
-
 	err = yaml.Unmarshal(data, courseFile)
 	if err != nil {
-		return nil, fmt.Errorf("failed to unmarshal file: %s", err.Error())
+		klog.V(3).Infof("failed to unmarshal file: %s", err.Error())
```

Comment:
Because I standardized on a single error message via the `SchemaValidationError` error type I provide more context to the underlying issue with a higher verbosity logging.

Reply:
👍🏼